sectionproperties/pre/library/concrete_sections.py

```diff
@@ -33,12 +36,17 @@ def concrete_rectangular_section(
     :param int n_bot: Number of bottom steel reinforcing bars
     :param int n_circle: Number of points discretising the steel reinforcing bars
     :param float cover: Side and bottom cover to the steel reinforcing bars
+    :param float dia_side: If provided, diameter of the side steel reinforcing bars
+    :param int n_side: If provided, number of side bars either side of the section
     :param float area_top: If provided, constructs top reinforcing bars based on their
         area rather than diameter (prevents the underestimation of steel area due to
         circle discretisation)
     :param float area_bot: If provided, constructs bottom reinforcing bars based on
         their area rather than diameter (prevents the underestimation of steel area due
         to circle discretisation)
+    :param float area_side: If provided, constructs side reinforcing bars based on
+        their area rather than diameter (prevents the underestimation of steel area due
+        to circle discretisation)
     :param Optional[sectionproperties.pre.pre.Material] conc_mat: Material to
         associate with the concrete
     :param Optional[sectionproperties.pre.pre.Material] steel_mat: Material to
```

Comment:
the ValueError below this has a similar spelling mistake to that noted on added column section

Reply:
same also in `concrete_tee_section` method.